Library/Formula/bltool.rb

```diff
@@ -0,0 +1,27 @@
+class Bltool < Formula
+  desc "A tool for command-line interaction with backloggery.com"
+  homepage "https://github.com/ToxicFrog/bltool"
+  url "https://github.com/ToxicFrog/bltool/releases/download/v0.2.1/bltool-0.2.1.zip"
+  sha256 "dceb5812058699b726bcd43a87c703eba39527959f712ef262919e0e14d6608e"
+
+  head do
+    url "https://github.com/ToxicFrog/bltool.git"
+    depends_on "leiningen" => :build
+  end
+
+  def install
+    if build.head?
+      system "lein", "uberjar"
+      bltool_jar = Dir["target/bltool-*-standalone.jar"][0]
+    else
+      bltool_jar = "bltool.jar"
+    end
+
+    libexec.install bltool_jar
+    bin.write_jar_script libexec/File.basename(bltool_jar), "bltool"
+  end
+
+  test do
+    assert_match /backloggery/, shell_output("#{bin}/bltool --help formats")
```

Comment:
Can this test be modified to do something more substantial than e.g. `--version` or `--help`? See `cmake.rb` for an example of an application formula with a good test and `tinyxml2.rb` for an example of a library formula with a good test. Thanks!